components/user/usermodel.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+
+const bcrypt = require('bcrypt');
+
+const SALT_ROUNDS = 10;
+
+const { Schema } = mongoose;
+
+const userSchema = new Schema({
+  username: { type: String, required: true },
+  password: { type: String, required: true },
+  email: { type: String, required: true },
+}, {
+  timestamps: {
+    createdAt: 'date_joined',
```

Comment:
I'm not asking for it to be changed but I'm wondering why you went for underscore naming rather than camelCase.